arch/AArch64/AArch64InstPrinter.c

```diff
@@ -2070,7 +2076,7 @@ void printAdrpLabel(MCInst *MI, uint64_t Address, unsigned OpNum, SStream *O)
 		return;
 	}
 
-	SStream_concat0(O, "<llvm-expr>");
+	printUInt64Bang(O, MCOperand_getImm(Op));
```

Comment:
Interesting catch. I wonder how it worked before.

Reply:
This was new and still works either way. I added it there a few weeks ago to show the user that the operand get's processed somehow by LLVM.

But in practice it is barely reached and it turns out LLVM just prints 0 in this case. So here it is reverted back to this behavior.